resources/views/projects/details.blade.php

```diff
@@ -90,7 +90,74 @@
 					@endforeach
 				</div>
 			</div>
+			<!-- Progress section -->
+			<div class="sixteen wide column">
+				<div class="ui left aligned detail-container">
+					<p><strong>Progress</strong></p>
+					<div class="ui blue progress">
+						<div class="bar">
+							<div class="progress"></div>
+						</div>
+						<div class="label">Currently on: Design</div>
+					</div>
+					<div class="ui right floated button" onclick="displayMilestones()">
+						<a class="milestones" title="See project's milestones">See milestones</a>
+					</div>
+					<div class="ui list milestone-section hidden">
+						<div class="item done">
+							<i class="large circle green icon"></i>
+							<div class="content">
+								<div class="header">Ideation</div>
+								<div class="description">Finished on 4/20</div>
+							</div>
+						</div>
+						<div class="item current">
+							<i class="large circle blue icon"></i>
+							<div class="content">
+								<div class="header">Design</div>
```

Comment:
Why are these `div`s? This is text

Reply:
That's how semantic ui works: https://semantic-ui.com/collections/message.html